src/corelibs/U2Core/src/datatype/primers/PrimerStatistics.cpp

```diff
@@ -316,10 +314,12 @@ QString PrimersPairStatistics::generateReport() const {
     result += "</tr>";
 
     QString e;
-    CREATE_ROW("% GC", GC_RANGE, toString(forward.getGC()), toString(reverse.getGC()), forward.isValidGC(e), reverse.isValidGC(e));
+    CREATE_ROW("GC (%)", GC_RANGE, toString(forward.getGC()), toString(reverse.getGC()), forward.isValidGC(e), reverse.isValidGC(e));
     CREATE_ROW(TmString, TM_RANGE, toString(forward.getTm()), toString(reverse.getTm()), forward.isValidTm(e), reverse.isValidTm(e));
-    CREATE_ROW("GC Clamp", CLAMP_RANGE, QString::number(forward.getGCClamp()), QString::number(reverse.getGCClamp()), forward.isValidGCClamp(e), reverse.isValidGCClamp(e));
-    CREATE_ROW("Runs", RUNS_RANGE, QString::number(forward.getRuns()), QString::number(reverse.getRuns()), forward.isValidRuns(e), reverse.isValidRuns(e));
+    const QString clumpRange = PrimerStatistics::tr("&gt;=%1 G or C at 3' end").arg(PrimerStatisticsCalculator::CLAMP_BOTTOM);
+    CREATE_ROW(PrimerStatistics::tr("GC Clamp"), clumpRange, QString::number(forward.getGCClamp()), QString::number(reverse.getGCClamp()), forward.isValidGCClamp(e), reverse.isValidGCClamp(e));
```

Comment:
GC Clamp and Poly-X are standart namings of these types of features in PCR design (e.g. primer3 has tem either). But I just have no idea how to translate Poly-X